packages/calypso-e2e/src/lib/blocks/block-flows/layout-grid.ts

```diff
@@ -73,23 +76,26 @@ export class LayoutGridBlockFlow implements BlockFlow {
 		columnDetails: ColumnDetails,
 		context: EditorContext
 	): Promise< void > {
-		const popoverBlockInserter = new PopoverBlockInserterComponent(
-			context.editorIframe,
-			context.page
-		);
+		const openInlineInserter: OpenInlineInserterDelegate = async ( editor: Locator ) => {
```

Comment:
I can see myself or others being confused by the use of Delegate here, and in `EditorPage`. Perhaps a short comment explaining our intent in either location would be great to have.

Reply:
Sure thing! I'll put comments in the scripts themselves.

As a quick gut check -- do you think it's too confusing to even take a delegate as a parameter?

I waffled back and forth a lot here... Adding blocks inline is tricky, because there's so many different ways to launch the inserter! And you can't even reliably guarantee there to be only one add block button on the page 😆 .

However, that makes the `addBlockInline` method notably different from `addBlockFromSidebar`, which can safely take care of launching the sidebar. I was trying to find the most obvious way to make that distinction clear, and adding a required delegate to the signature seemed the most intrusive, but also safest, route.

Reply:
I think I was being more confused by the term "Delegate" more than anything. I had a very superficial understanding of what a delegate does in the software context, but have never encountered it or knowingly used it. 

To make sure I understand correctly here: the delegate here refers to any inline block inserter, be it on the main document or within a block. The idea of this is to say that 'whatever happens, something that implements an inline inserter will be passed in here'. Is that more or less correct?

Re: the challenges of the Inline inserter - this is definitely an area that warrants further investigation if we continue developing increasing complex tests.

Reply:
Ahh, after some internet research, I'm now realizing that "delegate" is a term I picked up from my old c# days, and that it's not as universal a term! 

A "degelate" is just a function passed as a parameter! So, a callback basically (but a callback kind of implies that the function is called at the end).

I'm going to remove that verbiage. Sorry for the confusion!